app/models/miq_widget.rb

```diff
@@ -472,7 +472,7 @@ def self.sync_from_hash(attrs)
 
     widget = find_by(:description => attrs["description"])
     if widget
-      if filename && widget.updated_at.utc < File.mtime(filename).utc
+      if filename
```

Comment:
Same as above.  I don't think this will break anything but it's been a while since I looked at how we query reports and widgets to know if we ever sort by updated_on

Reply:
It looks like list of widget sorted alphabetically and not changing between server reboots